src/core/sys/posix/time.d

```diff
@@ -126,8 +126,6 @@ else version (Solaris)
 else version (CRuntime_Bionic)
 {
     enum CLOCK_MONOTONIC        = 1;
-    enum CLOCK_MONOTONIC_RAW    = 4;
-    enum CLOCK_MONOTONIC_COARSE = 6;
```

Comment:
In fact those constants are a kernel property and should merged under `version (linux)`.
https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/unix/sysv/linux/clock_getres.c;hb=b2fb25240813266e4f62f2dd039573f1a042ea2f#l39